pkg/builder/webhook_test.go

```diff
@@ -419,157 +361,25 @@ func runTests(admissionReviewVersion string) {
 		ExpectWithOffset(1, w.Body).To(ContainSubstring(`"message":"panic: fake panic test [recovered]`))
 	})
 
-	It("should scaffold a validating webhook with a custom validator", func() {
```

Comment:
Is this test case lost?

Sorry it's really had to diff. Can you please check the diff and change the ordering in your new version a bit to get a minimal diff? (it's really had because it looks like some test cases were moved around)

Reply:
Yeah this moved to line 219.  I can try to minimize the diff but deleting tests are going to blob where it's red.  I'll see what I can do.

Reply:
Yup that's fine. I'm just currently shuffling tests around to see what the actual diff is :D

Reply:
Sounds good, no worries. Thx!

Reply:
Reviewed, thank you very much! A lot easier to tell now that everything is fine :)